website/src/components/genspectrum/GsMutationsOverTime.tsx

```diff
@@ -36,7 +45,11 @@ export const GsMutationsOverTime: FC<GsMutationsOverTimeProps> = ({
                 granularity={granularity}
                 lapisDateField={lapisDateField}
                 displayMutations={displayMutations ? JSON.stringify(displayMutations) : undefined}
+                useNewEndpoint={useNewEndpoint ?? false}
```

Comment:
Same trap with boolean attributes: If we write `false` here, then the attibute in the HTML will be `usenewendpoint="false"`, which will result in he value being `true` in the component.
```suggestion
                useNewEndpoint={useNewEndpoint}
```

Using `undefined` for the "false" case works (React will simply omit the attribute, which is what we want)